static/js/tippyjs.js

```diff
@@ -449,20 +449,35 @@ export function initialize() {
     });
 
     delegate("body", {
-        target: "#show_all_private_messages",
+        target: ["#show_all_private_messages", ".message_expander", ".message_condenser"],
         placement: "bottom",
-        content: $t({
-            defaultMessage: "All private messages (P)",
-        }),
+        onShow(instance) {
+            const $elem = $(instance.reference);
+            const $template = $("#" + $elem.attr("data-tooltip-template-id"));
+            instance.setContent(parse_html($template.html()));
+        },
         appendTo: () => document.body,
     });
 
     delegate("body", {
         target: ".view_user_card_tooltip",
-        content: $t({
-            defaultMessage: "View user card (u)",
-        }),
+        onShow(instance) {
+            const $elem = $(instance.reference);
+            const $template = $("#" + $elem.attr("data-tooltip-template-id"));
+            instance.setContent(parse_html($template.html()));
+        },
         delay: [500, 20],
         appendTo: () => document.body,
     });
+
+    delegate("body", {
```

Comment:
Since the config for this block is the same as the one used for the block with `target: "#show_all_private_messages"`, what do you think about merging these two blocks together?

cc: @timabbott @sayamsamal

Reply:
Thank you for reviewing my work and providing valuable feedback. I appreciate your attention to detail and your suggestions for improvement. I will definitely take your recommendations into consideration and make the necessary updates.